tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraFilterSetMethodContract.scala

```diff
@@ -110,6 +110,16 @@ trait LinagoraFilterSetMethodContract {
          |					"id": "singleton",
          |					"rules": [{
          |						"name": "My first rule",
+         |						"conditionGroup": {
```

Comment:
In /set we should be able to specify `conditionGroup` OR (exclusive) `condition`

=> If both then throw.

Reply:
"If both then throw" => Fixed in FilterSet file line 57

both tests have already existed